stateproof/builder.go

```diff
@@ -357,29 +357,8 @@ func (spw *Worker) broadcastSigs(brnd basics.Round, proto config.ConsensusParams
 	}
 }
 
-func lowestRoundToRemove(currentHdr bookkeeping.BlockHeader) basics.Round {
-	proto := config.Consensus[currentHdr.CurrentProtocol]
-	nextStateProofRnd := currentHdr.StateProofTracking[protocol.StateProofBasic].StateProofNextRound
-	if proto.StateProofInterval == 0 {
-		return nextStateProofRnd
-	}
-
-	recentRoundOnRecoveryPeriod := basics.Round(uint64(currentHdr.Round) - uint64(currentHdr.Round)%proto.StateProofInterval)
-	oldestRoundOnRecoveryPeriod := recentRoundOnRecoveryPeriod.SubSaturate(basics.Round(proto.StateProofInterval * proto.StateProofMaxRecoveryIntervals))
-	// we add +1 to this number since we want exactly StateProofMaxRecoveryIntervals elements in the history
-	oldestRoundOnRecoveryPeriod++
```

Comment:
difference with the new code: oldestRoundOnRecoveryPeriod is not incremented in the new